src/businessEvents/unifiedEventKinesisHandler.ts

```diff
@@ -109,12 +110,17 @@ export async function unifiedEventKinesisHandler(
       },
       []
     );
-    const errorMessage = `ERROR: Failed to send ${
-      failedEvents.length
-    } event(s) to kinesis stream '${
-      config.aws.kinesis.unifiedEvents.streamName
-    }'. Failed Events: \n ${JSON.stringify(failedEvents)}`;
-    console.log(errorMessage);
-    Sentry.captureException(new Error(errorMessage));
+    serverLogger.error('Failed to send event(s) to kinesis stream', {
+      stream: config.aws.kinesis.unifiedEvents.streamName,
+      events: JSON.stringify(failedEvents),
+    });
+    Sentry.addBreadcrumb({
```

Comment:
maybe a tangent - did we figure out @efixler's questions about following breadcrumbs? just wondering how useful this is with `ts-logger` in place.

Reply:
Yea. I think long term, ts-logger needs to emit to Sentry breadcrumbs, but we just haven't set a pattern on making that happen.